gp-templates/locales.php

```diff
@@ -1,8 +1,7 @@
 <?php
 gp_title( __( 'Locales &lt; GlotPress', 'glotpress' ) );
 
-gp_enqueue_script( 'gp-common' );
-gp_enqueue_script( 'tablesorter' );
+gp_enqueue_script( array( 'gp-common', 'gp-editor', 'tablesorter' ) );
```

Comment:
It threw a JavaScript error on the page if it wasn't included, I didn't track it down any farther than that thought.